src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepositorySpec.scala

```diff
@@ -0,0 +1,36 @@
+package uk.gov.nationalarchives.tdr.api.db.repository
+
+import java.sql.Timestamp
+import java.time.{Instant, LocalDateTime}
+import java.util.UUID
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import uk.gov.nationalarchives.Tables.ConsignmentRow
+import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import uk.gov.nationalarchives.tdr.api.utils.TestUtils
+import uk.gov.nationalarchives.tdr.api.utils.TestUtils._
+
+import scala.concurrent.ExecutionContext
+
+class ConsignmentRepositorySpec extends AnyFlatSpec with ScalaFutures with Matchers {
+  implicit val executionContext: ExecutionContext = ExecutionContext.Implicits.global
+
+  "addParentFolder" should "add parent folder name to an existing consignment row" in {
+    val db = DbConnection.db
+    val consignmentRepository = new ConsignmentRepository(db)
+    val consignmentId = UUID.fromString("0292019d-d112-465b-b31e-72dfb4d1254d")
+
+    TestUtils.createConsignment(consignmentId, userId)
+    TestUtils.createFile(UUID.fromString("813231b3-2fbd-4869-abdf-0c42624c4d57"), consignmentId)
```

Comment:
Are these files necessary for this test? At a high level, we know that adding files happens at the same time as adding a folder name, but at a low level, the `addParentFolder` isn't related to the files at all.

Reply:
I hadn't thought of that actually! I've removed it and the test functions the same so I think it's best to just take them out since it doesn't really serve any purpose :+1: